codemetrics.el

```diff
@@ -317,13 +317,13 @@ For argument DEPTH, see function `codemetrics-analyze' for more information."
       '(0 nil))
     '(1 nil)))
 
-(defun codemetrics-rules--method-declaration (node depth nested)
-  "Define rule for `method' declaration.
+(defun codemetrics-rules--method-declaration-using-identifier-name (node depth nested identifier-name)
```

Comment:
Maybe rename this function since there is no real "identifier-name " is TreeSitter. 🤔

Reply:
Naming is hard 🙁 `node-name` probably makes more sense in tree-sitter terms?

Reply:
Yeah, that's much better!